downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReleaseTarball.java

```diff
@@ -0,0 +1,99 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.DirectoryFileFilter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+public class CreateReleaseTarball {
+	private static final Logger logger = LogManager.getLogger();
+	public static boolean runPerl = true;
+	
+	public static void execute(String releaseNumber, String releaseDownloadDir) throws IOException, InterruptedException {
+		logger.info("Running CreateReleaseTarball step");
+		
+		if (runPerl) {
+			String tarballCommand = "perl " + releaseDownloadDir + "/make_release_tarball.pl " + releaseNumber;
+			Process runPerlTarballCommand = Runtime.getRuntime().exec(tarballCommand);
+			runPerlTarballCommand.waitFor();
+		} else {
+		
+		
+			String tarDirRelease = "reactome_tar/" + releaseNumber + "/Release";
+			String absReleaseDir = "/usr/local/gkb/scripts/release/website_files_update";
+			String absReactomeDir = "/usr/local/reactomes/Reactome/production/Website/static/download/" + releaseNumber;
+			releaseNumber = releaseNumber.replace("/", "");
+			//TODO: When the dust has settled, make sure the tarball contains everything we need
+			
+			// Remove any existing Release repositorys
+			File releaseDir = new File(tarDirRelease);
+			releaseDir.mkdirs();
+			if (releaseDir.list().length > 0) {
+				Process removeReleaseRepo = Runtime.getRuntime().exec("rm -r " + tarDirRelease);
+				removeReleaseRepo.waitFor();
+			} 
+			logger.info("Cloning Release: https://github.com/reactome/Release.git...");
+			try {
+				Git.cloneRepository().setURI("https://github.com/reactome/Release.git").setDirectory(Paths.get(tarDirRelease).toFile()).call();
+				logger.info("Finshed cloning Release repository");
+			} catch (GitAPIException e) {
+				e.printStackTrace();
```

Comment:
I think log4j can capture exceptions in logger calls.  It might be a good idea to log any exceptions when they occur so we have them in a file.